DoLah/Commands/Command.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include "..\Models\Task.h"
+
+namespace DoLah {
+
+	class Calendar;
+
+	class AbstractCommand {
+	public:
+		AbstractCommand();
+		virtual ~AbstractCommand() = 0;
+
+		void setCalendar(Calendar* cal);
+		Calendar* getCalendar();
+
+		void execute();
```

Comment:
To ensure that the inherited class could execute its respective function during the polymorphic behavior, please consider making this to be a virtual function.